### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by robruiz** [wprig/wprig#738] [General]
Introduction of new base customizer settings class and JSON pattern for controlling them

<hr /><em>This discussion was created from the release <a href='https://github.com/wprig/wprig/releases/tag/2.1.0'>v2.1.0</a>.</em>

**Comment by ghost** [comment]
Is there some kind of documentation for upgrading from v1 to v2? I have one live site and another in staging. Thanks for your help and valuable work supporting the project. :)

**Reply by LukaszJaro** [reply]
I wouldn't even try such a thing, v1 and v2 are different animals, many differences between them like different file structure. Also I don't think that's how wprig was meant to be used.

**Comment by ghost** [comment]
Thanks for the update. I will use the new version on an upcoming project.


*Reuben Walker, Jr.*




*Mobile Atom Media*
https://media.mobileatom.net/


*Mobile Atom Code*https://www.mobileatom.net/
*Currently mastering Drupal, PHP 8.0, & the Symfony Framework
<https://symfonycasts.com/u/mobileatom>*

*Symfony Station*
https://symfonystation.com/

*Stack Overflow Story*

https://stackoverflow.com/story/mobileatom

*Content Marketing, Production, and Maintenance for Small Organizations
with Big Ideas*

   - Follow me on Twitter for curated best practices in content marketing
   and coding <https://twitter.com/mobileatom>.
   - Let's connect on LinkedIn so you can tap into my 600+ connections
   <https://www.linkedin.com/in/reubenwalker>.
   - Interested in individual marketing, design, content, and coding
   topics? Join 43K+ followers of my Flipboard Magazines
   ***@***.***>.

**Comment by LukaszJaro** [comment]
Actually @robruiz, what is the best practice to upgrade v2.0?

**Comment by robruiz** [comment]
I would recommend just creating a duplicate environment on your local
machine, upgrade, see what happens. If problems arise, assess the situation
and solve any problems that come up. If there are too many, you might be
better off just rebuilding and copying/pasting as much code as possible.

On Wed, Dec 8, 2021 at 7:15 AM LukaszJaro ***@***.***> wrote:

> Actually @robruiz <https://github.com/robruiz>, what is the best practice
> to upgrade v2.0?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wprig/wprig/discussions/738#discussioncomment-1772520>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6LMGJCVLFRAUMC6B4CZXDUP5LAZANCNFSM5BQHMSJA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Rob Ruiz
Owner of Bizzle Designs
Web Design/Graphic Design/Sound Design/Audio & Video Production
www.bizzledesigns.com

**Reply by LukaszJaro** [reply]
Thanks!, that's what I thought, the duplicate environment is a good idea.
